lms/djangoapps/teams/serializers.py

```diff
@@ -87,6 +93,10 @@ def restore_object(self, attrs, instance=None):
 
 class MembershipSerializer(serializers.ModelSerializer):
     """Serializes CourseTeamMemberships with information about both teams and users."""
+    profile_configuration = deepcopy(settings.ACCOUNT_VISIBILITY_CONFIGURATION)
```

Comment:
It seems that all of these expansion serializers need a way to inject the URL into their response. Would it be possible to do it with a mixin? Another option would be to have the ``ExpandableField`` automatically do this to the serialized form so that every field's serializer doesn't have to implement it (and might forget to do so).